hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/mod_NERSCnml.F90

```diff
@@ -46,8 +48,9 @@ subroutine NERSC_init
     end do
     close(funi)
     if (mnproc.eq.1) then
-      write (lp,*)'NERSC HYCOM: Reading hycom_nml from: hycom_opt'
+      write (lp,*)'NERSC HYCOM: Reading hycom_nml from: ../hycom_opt'
```

Comment:
I assume that this means that hycom_opt is located in the directory above SCRATCH (fine by me).

Reply:
I try to keep the same location as used by hycom_fabm.